script.js

```diff
@@ -269,9 +269,53 @@ const loadWorks = (data) => {
 
 window.addEventListener('DOMContentLoaded', () => {
   loadWorks(data);
+  const emailData = window.localStorage.getItem('emailData');
+  const messageData = window.localStorage.getItem('messageData');
+  const nameData = window.localStorage.getItem('nameData');
+  if (emailData) {
+    const email = document.querySelector('.email-area');
+    email.value = emailData;
+  }
+  if (messageData) {
+    const message = document.querySelector('.text-area');
+    message.value = messageData;
+  }
+  if (nameData) {
+    const fullName = document.querySelector('.name-area');
+    fullName.value = nameData;
+  }
 
   const loadModalEle = document.querySelector('#loadModal');
   loadModalEle.addEventListener('click', () => {
     loadModal(loadModalEle.value);
   });
-});
\ No newline at end of file
+});
+
+// form validation
+
+const form = document.querySelector('form');
+const alertMsg = document.querySelector('small');
+
+const email = document.querySelector('.email-area');
+const message = document.querySelector('.text-area');
+const fullName = document.querySelector('.name-area');
+email.addEventListener('change', () => {
+  window.localStorage.setItem('emailData', email.value);
+});
+message.addEventListener('change', () => {
+  window.localStorage.setItem('messageData', message.value);
+});
+fullName.addEventListener('change', () => {
+  window.localStorage.setItem('nameData', fullName.value);
+});
```

Comment:
- [ ] Your storage is working fine, it adds to local storage and can read data from there to populate the form correctly. But as the project required, you should store the data into a single Javascript object instead of creating 3 different fields in the local storage. Kindly change that to meet the project requirements and be aware to change the code that reads the data to populate the UI too.
 - 
<img width="1000" alt="image" src="https://user-images.githubusercontent.com/61919617/216318518-0870a4c6-0d74-4de3-aac6-3c0e0b029512.png">

Reply:
Thanks for the review @leonardodiasb